src/cli_plugin/install/__tests__/kibana.js

```diff
@@ -45,7 +45,7 @@ describe('kibana cli', function () {
             tempArchiveFile: tempArchiveFilePath,
             plugin: 'test-plugin',
             version: '5.0.0-SNAPSHOT',
-            plugins: [ { name: 'foo', path: join(testWorkingPath, 'foo'), version: '5.0.0-SNAPSHOT' } ]
+            plugins: [ { name: 'foo', path: join(testWorkingPath, 'foo'), kibanaVersion: '5.0.0-SNAPSHOT' } ]
```

Comment:
Shouldn't the tests be checking both `version` and `kibanaVersion`? As in, one set for only `kibanaVersion` and another for `version`?